src/python/pathops/_pathops.pxd

```diff
@@ -102,7 +102,7 @@ cdef class Path:
     @staticmethod
     cdef Path create(const SkPath& path)
 
-    cpdef PathPen getPen(self, bint allow_open_paths=*)
+    cpdef PathPen getPen(self, object glyphSet=*, bint allow_open_paths=*)
```

Comment:
I used camelCase for glyphSet because that's how it is traditionally called in fonttools pens API